play.go

```diff
@@ -10,6 +10,7 @@ func (game *Game) play() {
 	game.gameOn = true
 	for game.gameOn == true {
 		piecePlaced := false
+		playerPassed := false
```

Comment:
what if, for example, light passes, dark does not, then light passes again?
won't that incorrectly end the game?
I think `placePiece()` should set `playerPassed` back to `true` over and over again.
Also, maybe we should store `playerPassed` on the game object itself.